Wrappers/Python/cil/plugins/astra/processors/AstraBackProjector2D.py

```diff
@@ -85,6 +85,19 @@ def check_input(self, dataset):
             raise ValueError("Expected input dimensions is 1 or 2, got {0}"\
                              .format(dataset.number_of_dimensions))
 
+    def check_output(self, out):
```

Comment:
does `check_output` need redefining? I think you could just set `self._shape_out` to `self.volume_geometry.shape` in the `init`.

Reply:
Good point! _shape_out is set in framework with _set_up so I can't set it in the init but I've redefined _set_up here instead.